.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: build
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: checkout-sp
```

Comment:
This and a couple other instances should be `wrf_io`

Reply:
Done.

Reply:
I don't see any commits

Reply:
Sorry, typo in the git command. I just committed these changes...